config/routes.rb

```diff
@@ -25,8 +25,12 @@
       get :wizard_install
     end
   end
-  resources :users, only: [:index, :edit]
-  resources :invitations, only: [:new, :show]
+  resources :users, only: [:index, :edit, :update] do
+    member do
+      get :confirm_account
```

Comment:
you can write this as `get :confirm` as there is no need for `confirm_account`. It's understandable.